platform/security/servlet/security-servlet-logout/src/test/java/org/codice/ddf/security/servlet/logout/TestLogoutServlet.java

```diff
@@ -33,40 +33,81 @@
 import javax.servlet.http.HttpSession;
 import org.apache.shiro.subject.Subject;
 import org.apache.shiro.util.ThreadContext;
+import org.junit.Before;
 import org.junit.Test;
 
 public class TestLogoutServlet {
-  @Test
-  public void testLocalLogout() {
+  LocalLogoutServlet localLogoutServlet;
+  HttpServletRequest request;
+  HttpServletResponse response;
+  HttpSession httpSession;
+
+  @Before
+  public void testsetup() {
```

Comment:
`setUp`

Reply:
Naming it setup actually gave me issues, I was getting some odd errors that went away when I changed this 👐